firmware/shared/src/hw/hw_spi.c

```diff
@@ -12,7 +13,7 @@ void hw_spi_setNssHigh(const SpiInterface *spi)
     HAL_GPIO_WritePin(spi->nss_port, spi->nss_pin, GPIO_PIN_SET);
 }
 
-bool hw_spi_transmitAndReceive(
+bool hw_spi_transmitThenReceive(
```

Comment:
More accurate function description, it does not receive while command is being sent, so it is more of a sequential Transmit then Receive